app/controllers/guests_controller.rb

```diff
@@ -0,0 +1,16 @@
+class GuestsController < ApplicationController
+
+  def show
+    @guest = Guest.find(params[:id])
+    @rooms_stayed_in = @guest.rooms
+  end
+
+  def create
+    @guest = Guest.find(params[:id])
+    room = Room.find(params[:guest][:room_id])
+    
+    @guest.rooms << room
+
+    redirect_to "/guests/#{@guest.id}"
+  end
```

Comment:
Since you're creating a GuestRoom resource, this should be in a GuestRoomsController. 

Also, we could make this code a bit more optimal by creating the resource right away, rather than finding two objects and making an association. It could look something like this: 

```ruby
    GuestRoom.create(guest_id: params[:id], room_id: params[:room_id])
    redirect_to "/guests/#{params[:id]}"
```